packages/verifiable-credentials/src/credentials.ts

```diff
@@ -169,8 +175,14 @@ export default class Credentials {
 		// Ensure data matches specified schema
 		const schema = await this.context.getClient().getSchema(data.schema)
 		const isValid = await schema.validate(data);
+		const schemaJson = await schema.getSpecification();
+		const databaseName = schemaJson['database']['name']
 
-		// @todo: Check the schema is a "credential" schema type?
+		if (schemaJson && databaseName === 'credential') {
+			if (!data.didJwtVc) {
```

Comment:
@cmcWebCode40 You need to check that `schemaJson` has the `didJwtVc` attribute, not that the attribute is set in the credential data.

You are checking the schema has the property, not that the data matches the schema.

See the comment you deleted:

`@todo: Check the schema is a "credential" schema type?`

Reply:
This also needs a unite test.